enhancing/enhancer.js

```diff
@@ -2,19 +2,39 @@ module.exports = {
   succeed,
   fail,
   repair,
-  get,
+  get
 };
 
 function succeed(item) {
-  return { ...item };
+  if (item.enhancement === 20) {
+    return { ...item };
+  } else {
+    item.enhancement += 1;
+    return { ...item };
+  }
 }
 
 function fail(item) {
+  if (item.enhancement < 15) {
+    item.durability -= 5;
+  } else if (item.enhancement > 15) {
```

Comment:
```suggestion
  } else if (item.enhancement >= 15) {
```
It wants 15 and greater